pom.xml

```diff
@@ -65,9 +65,11 @@
   </scm>
 
   <properties>
-    <jenkins.version>1.625.3</jenkins.version>
-    <java.level>7</java.level>
+    <jenkins.version>2.60.3</jenkins.version>
+    <java.level>8</java.level>
     <antlr4.version>4.5</antlr4.version>
+    <jcasc.version>1.13</jcasc.version>
+    <access-modifier-checker.failOnError>false</access-modifier-checker.failOnError>
```

Comment:
I would like to avoid this :sweat: Can we remove the XML parsing?

Reply:
> Can we remove the XML parsing?

Context?

Reply:
I didn't look too closely. It seems that credentials allow you to send both JSON and XML requests to create credentials.

Reply:
Which API are you using that's prohibited to plugins?

Reply:
XMLUtils is marked restricted and is used here:
https://github.com/jenkinsci/credentials-plugin/blob/6f710a7e30156e94aff8a3c6c6799cef04ad35df/src/main/java/com/cloudbees/plugins/credentials/CredentialsStoreAction.java
https://github.com/jenkinsci/credentials-plugin/blob/6f710a7e30156e94aff8a3c6c6799cef04ad35df/src/main/java/com/cloudbees/plugins/credentials/cli/BaseCredentialsCLICommand.java

Reply:
Ah, OK, it used to be available due to a bug in the access modifier checker used in old parent POMs. We're not updating that, so can no longer use it by default. That explains why it didn't show in the diff.

Reply:
So your saying we should keep the code as is?
That would mean that this code is ready for merging I'd guess.